specs/trs/tracked-resource-set.html

```diff
@@ -423,15 +423,28 @@ <h2>Base Segmentation</h2>
       <p>
         The number of Tracked Resources in the Base could be such that it is not reasonable to list them all in a single
         HTTP response. In this case,
-        <span class="conformance"
-          >the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
-          listing a subset of all Tracked Resources</span
-        >.
+        <span class="conformance">
+          the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
+          listing a subset of all Tracked Resources.</span
+        >
+      </p>
+      <p class="conformance">
+        If base paging is implemented, the server MAY respond with a 30x redirect message, directing the Client to the
+        first “single-page resource”, or alternatively it MAY respond with the first "single-page resource" .
       </p>
       <p class="conformance">
-        Base segmentation MUST be accomplished using the mechanisms described in the "Resource Paging" section of
-        [[!OSLCCore3]].
+        The representation of a single-page resource MAY contain a subset of the Base’s membership triples.
```

Comment:
How does this map to "7.1.1 LDP Paging servers MUST ensure that the membership triple and containment triple for each member are part of the same in-sequence page resource, whenever both triples are present in the page sequence for a paged LDPC."?

I feel that:

1. OSLC MAY and LDP MUST are in conflict here.
2. "subset of the Base’s membership triples" is too vague.

Reply:
We are not implementing or referencing LDP paging. Forget that LDP paging exists. It's not relevant here.